tools/deljobs.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+for j in $(kubectl get jobs -o custom-columns=:.metadata.name)
```

Comment:
Where is this script getting invoked from?

Reply:
Since this is just generic - getting the jobs from kubectl and removing them - it's really just a cleanup script. I can remove this if it's not useful.

Reply:
Yes, lets remove this if its not being used in the e2e tests. It may be useful to get some e2e tests created as well (as a part of a seperate pull request).

Reply:
It's gone